.gitignore

```diff
@@ -6,3 +6,5 @@ build/
 
 crates/pill_standalone/Cargo.toml
 examples/*/Cargo.toml
+
+engine/Cargo.toml

```

Comment:
I know why you are doing that, though this is probably not the best solution 😓 

The launcher will constantly update the paths in `engine/Cargo` and show this file as changed. Honestly did not find a good solution for that except calling `git add engine/pill_*` when staging files.